coffeeshop-service/src/main/webapp/index.html

```diff
@@ -130,7 +130,7 @@ <h1>Queue</h1>
     });
 
     $(function () {
-        var source = new EventSource(window.location.protocol + "//" + window.location.host + "/queue");
+        var source = new EventSource("http://localhost:8080/services/queue");
```

Comment:
This will only work if you are running locally on 8080, you probably want:
```
var source = new EventSource(window.location.protocol + "//" + window.location.host + "/services/queue");
```

Reply:
Oops, I reverted too much, thanks for spotting.